### PR TITLE
PS-1466 LP #1264306: Per Session innodb_flush_log_at_trx_commit poorly documented (8.0)

### PR DESCRIPTION
**Pull request by patrickbirch**
8.0 version
file modified: doc/source/scalability/ innodb_io.rst

### CODE REVIEW

**Comment by patrickbirch** [comment]
waiting for comments or approval.

**Review by dutow — CHANGES_REQUESTED** [review]
This variable doesn't exist in the 8.0 branch, it was removed during the 8.0 port.

The jira ticket requests changes to the already existing 5.7 documentation.

**Pull request closed without merging**
